src/frontend/efiling-frontend/src/domain/package/package-review/PaymentListService.js

```diff
@@ -10,5 +10,5 @@ export const downloadPaymentReceipt = async (packageId) => {
   );
   const fileData = new Blob([response.data], { type: "application/pdf" });
   const fileUrl = URL.createObjectURL(fileData);
-  FileSaver.saveAs(fileUrl, "SubmissionSheet.pdf");
+  FileSaver.saveAs(fileUrl, "PaymentReceipt.pdf");
```

Comment:
is the name not provided in the header of the response ?

Reply:
From what I can see, no part of the response contains the filename. Is there a specific naming convention that is preferred for these file names?